File: algorithm/typescript/BOJ10952.ts
// problem : https://www.acmicpc.net/problem/10952

import fs = require("fs");

const input: string[] = fs.readFileSync("/dev/stdin").toString().split("\n");
const answer: number[] = [];

for (let i = 0; i < input.length; i++) {
  const numbers: number[] = input[i].split(" ").map(Number);

  const A: number = numbers[0];
  const B: number = numbers[1];

  if (A == 0 && B == 0) {
    break;
  }

  answer.push(A + B);
}

console.log(answer.join("\n"));

File: algorithm/python/BOJ10952.py
# https://www.acmicpc.net/problem/10952

import sys

answer_list = list()

while True:
    A, B = map(int, sys.stdin.readline().rstrip().split())

    if A == 0 and B == 0:
        break

    answer_list.append(A + B)

for value in answer_list:
    print(value)
